sw/device/lib/dif/dif_plic.c

```diff
@@ -56,13 +57,15 @@ typedef struct plic_target_reg_offset {
 // - `RV_PLIC_IE<i>_0_REG_OFFSET` (the first IE reg for target `i`).
 // - `RV_PLIC_CC<i>_REG_OFFSET`
 // - `RV_PLIC_THRESHOLD<i>_REG_OFFSET`
+// - `RV_PLIC_MSIP<i>_REG_OFFSET`
 static const plic_target_reg_offset_t plic_target_reg_offsets[] = {
```

Comment:
nit: `kPlicTargetTegOffsets`

Reply:
Good point, however I think this one I will just fire separately.

Reply:
Fixed with @mcy recent naming refactor.